frontend/next-i18next.config.js

```diff
@@ -10,13 +10,10 @@ module.exports = {
     defaultLocale: 'default',
   },
   /** To avoid issues when deploying to some paas (vercel...) */
-  localePath:
-    typeof window === 'undefined'
-      ? require('path').resolve('./public/locales')
-      : '/locales',
-  returnNull: false,
+  localePath: typeof window === 'undefined' ? require('path').resolve('./public/locales') : '/locales',
   react: {
     transKeepBasicHtmlNodesFor: ['br', 'strong', 'i', 'p', 'b', 'em'],
   },
   reloadOnPrerender: process.env.NODE_ENV === 'development',
+  serializeConfig: false,
```

Comment:
When you do this, don't you also have to add the config to `appWithTranslation(..)`, too?

Reply:
It's already added.

Reply:
You also need to add it to `getServerSideProps(..)`, don't you?

Reply:
https://github.com/i18next/next-i18next#unserializable-configs

![image](https://github.com/DTS-STN/senior-journey/assets/48123208/3c089ccf-0e50-417d-8bec-b52b23862ba0)

Reply:
I didn't had too.

Reply:
> To fix this, you'll need to set config.serializeConfig to false, and manually pass your config into appWithTranslation:

Reply:
Should add it anyway.. future developers might modify the config and trigger the unserializable error.

Reply:
I will need to double check if it's really needed.

Reply:
It's not really needed.. until it is. I was just thinking about future developers and how this could break if they add a `onPreInitI18next` config option or something.

Reply:
ok, fair enough. Will follow the docs.